internal/ingress/controller/config/config.go

```diff
@@ -17,11 +17,10 @@ limitations under the License.
 package config
 
 import (
+	"k8s.io/klog/v2"
```

Comment:
Please rollback this change to the right place

Reply:
This was addressed!